cmd/context/create.go

```diff
@@ -108,22 +108,26 @@ func (c *ContextCommand) UseContext(ctx context.Context, ctxOptions *ContextOpti
 
 	ctxStore := okteto.ContextStore()
 	if okCtx, ok := ctxStore.Contexts[ctxOptions.Context]; ok && okCtx.IsOkteto {
+		ctxOptions.Namespace = okCtx.Namespace
 		ctxOptions.IsOkteto = true
 	}
 
 	if okCtx, ok := ctxStore.Contexts[okteto.AddSchema(ctxOptions.Context)]; ok && okCtx.IsOkteto {
-		ctxOptions.Context = okteto.AddSchema(ctxOptions.Context)
+		ctxOptions.Context = okteto.AddSchema(ctxOptions.Context) // Should this be changed to okCtx.Context?
+		ctxOptions.Namespace = okCtx.Namespace
```

Comment:
Maybe I should keep this line since `ctxOptions.Namespace = ctxStore.Contexts[ctxOptions.Context].Namespace` won't fit here.

Reply:
We must remove this line since `ctxOptions.Namespace` is the one that we one to preserve if it's set. If it's not set it will be set by default to use `ctxStore.Contexts[ctxOptions.Context].Namespace` so it's not needed here